README.md

```diff
@@ -2,7 +2,191 @@
 
 # Metamath set.mm repository
 
-This is a set of Metamath source files (databases) for logic and set theory.
+This is a collection of databases that specify mathematical axioms and
+rigorous formal proofs of various mathematical theorems
+derived from those axioms.
+This collection is specified and derived using the Metamath system.
+A "database" in Metamath parlance is a collection of one or more text files.
 
-File "set.mm" uses classical logic and ZFC set theory.
-File "iset.mm" uses intuitionistic logic.
+This collection is being actively improved by an international community.
+Please join us!
+
+The two largest databases in this collection are "set.mm" and "iset.mm".
+The "set.mm" database is one of the largest
+collections of formally-verified mathematics in the world (e.g., it
+has completed many challenge theorems in the
+[Formalizing-100 Theorems](https://www.cs.ru.nl/~freek/100/) challenge list).
+It is also one of the most rigorously verified collection of
+formally-verified mathematics in the world; it's verified by
+5 different verifiers written by 5 different people
+in 5 different programming languages.
+
+## What is Metamath?
+
+Metamath is a computer language and associated computer program for
+archiving, verifying, and studying mathematical proofs.
+
+Unlike some other systems, Metamath does not build a particular set
+of axioms into the system. Instead, the Matamath language is simple and robust,
+with an almost total absence of hard-wired syntax.
+In Metamath you express the axioms *and* the theorems in a database.
+To prove a theorem, every proof step *must* be proven using an axiom or
+a previously-proven theorem; as a result, nothing is hidden.
+We believe Metamath provides about the simplest possible framework that
+allows essentially of mathematics to be expressed with absolute rigor.
+
+For more information see
+the [Metamath Home Page](http://us.metamath.org/), the
+[Metamath Proof Explorer Home Page](http://us.metamath.org/mpeuni/mmset.html),
+the [Metamath book](http://us.metamath.org#book), or the
+[video "Metamath Proof Explorer: A Modern Principia Mathematica](https://www.youtube.com/watch?v=8WH4Rd4UKGE).
+
+## What databases are included in this collection?
+
+This is a collection of many databases, but since "set.mm" is by far
+the most actively maintained, you may want to look at it first even if you
+plan to eventually work on something else.
+[The video "Metamath Proof Explorer (set.mm) contributions visualized with Gource through 2019-10-04"]( https://www.youtube.com/watch?v=XC1g8FmFcUU&list=PL1jSu6GGefBm7RBP0Id2Sa9uyVuyhioAC&index=4) shows set.mm's growth over time.
+
+The databases included, in (approximate) decreasing size, are:
+
+* "set.mm" aka Metamath Proof Explorer (MPE) -
```

Comment:
To be consistent with the other entries, "Metamath Proof Explorer" should be in quotes.

Reply:
Agre.